device/imt/x86_64-im_n29xx_t40n-r0/default_sku

```diff
@@ -0,0 +1 @@
+im_t2624_40x t1
```

Comment:
what is t1?

Reply:
The default value for preset configuration. It means datacenter tier 1 and should give a hint to the SONiC OS where this device will be used. Based on this it can optimize resources allocation. It can be changed in run-time via sonic-cfggen utility. The other supported presets are "l2", "l3", "empty".